sdk/identity/azure-identity/tests/test_imds_credential.py

```diff
@@ -215,16 +216,20 @@ def test_imds_authority_override():
 
 
 @pytest.mark.usefixtures("record_imds_test")
-class RecordedTests(RecordedTestCase):
+class TestImds(RecordedTestCase):
+    @recorded_by_proxy
     def test_system_assigned(self):
         credential = ImdsCredential()
         token = credential.get_token(self.scope)
         assert token.token
         assert isinstance(token.expires_on, int)
+        return {}
```

Comment:
Good news! We can remove all of the `return {}` statements from test cases since the bug requiring this workaround has been fixed